library.go

```diff
@@ -213,10 +212,10 @@ func (lib *Library) ImportBook(book Book, move bool) error {
 		tx.Rollback()
 		return errors.Wrap(err, "Fetching new book ID")
 	}
-	bf.ID = id
+	book.Files[0].ID = id
```

Comment:
If bf is a pointer to book.files[0], can't we still use bf.id? Seems we can, but forgot if you tried that.